### DISCUSSION TITLE
<title> Copilot Pycharm partial suggestion acceptance

### DISCUSSION THREAD

**Discussion by ghostFaceKillah** [community/community#46093] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hello!

We would love to have https://github.com/microsoft/vscode/issues/140708 this, but in Pycharm 🙏🙏🙏🙏
Thanks!

I have seen this requested a couple of times around the forum, but mostly for VSCode. (duh! it's done folks!)

Goodbye

**Comment by petered** [comment]
Yo mike

**Comment by BoD** [comment]
See also #12426
